README.md

```diff
@@ -54,8 +54,8 @@ That's it! You are good to go.
 
 ## Caveats
 
-**IMPORTANT:** Be aware that `server-side` includes every first render of your application.
-Your secret tokens won't be included anywhere **except you use them **
+**IMPORTANT:** Be aware that `server` is included on the first render of your application.
+Your secret tokens won't be included anywhere **except where you use them **
```

Comment:
```suggestion
Your secret tokens won't be included anywhere **except where you use them**
```

Reply:
Ah, okay. I misunderstood how `server-side` was used, since it looked like it was referring to a variable/value.